.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,11 @@
-<!--
+## PR Checklist
 
-If you are a PowerShell Team member, please make sure you choose the Reviewer(s) and Assignee for your PR.
-If you are not from the PowerShell Team, you can leave the fields blank and the Maintainers will choose them for you. If you are familiar with the team, feel free to mention some Reviewers yourself.
+- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
+    - [ ] Use the present tense and imperative mood when describing your changes
+- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
+- [ ] [Update the changelog](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---changelog)
+- [ ] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission) (remove if not needed)
```

Comment:
Can we say 'insert NA' if not needed rather than removing?  Easier to catch if there's a disagreement.